static/static_data.json

```diff
@@ -1,9 +1,9 @@
 {
     "Orange": {
         "totalOldActive": 0,
-        "totalOldInactive": 48,
-        "totalNewDelivered": 104,
-        "totalNewUndelivered": 48
+        "totalOldInactive": 36,
```

Comment:
Was this number wrong?

Reply:
They've scrapped more cars. So there's less in the yard now